src/repairnator-jenkins-plugin/src/main/java/io/jenkins/plugins/main/Config.java

```diff
@@ -7,6 +7,8 @@
 	private static Config config;
 	private File tempDir;
 	private boolean quiet = false;
+	private boolean useTLS = false;
```

Comment:
is this information in Jenkins already?

Reply:
Unfortunately no, they only have useSSL .

Reply:
Not a problem, we don't need to do better than default Jenkins, so we can reuse their `useSSL` and remove the useTLS from here (field+g/setters+UI field)

Reply:
Fixed :) .